content/pt-br/docs/reference/glossary/ingress.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Ingress
+id: ingress
+date: 2018-04-12
+full_link: /docs/concepts/services-networking/ingress/
+short_description: >
+  Um objeto API que gerencia o acesso externo aos serviços em um cluster, normalmente HTTP.
+
+aka: 
+tags:
+- networking
+- architecture
+- extension
+---
+ Um objeto API (do inglês "Application Programming Interface") que gerencia o acesso externo aos serviços em um cluster, normalmente HTTP.
+
+<!--more--> 
+
+O Ingress pode fornecer um balanceamento de carga, descriptografia SSL e hospedagem virtual baseada em nome.
```

Comment:
This phrase is a bit strange. Maybe we don’t need the literal translation here and just say “Um ingress pode fornecer as capacidades de um proxy reverso para as aplicações”